### DISCUSSION TITLE
Move deploy settings to a separate section

### DISCUSSION THREAD

**Discussion by FransdeJong** [umbraco/Umbraco.Deploy.Issues#311] [Ideas]
I think it is time to move deploy settings away from the settings section to it's own section?
This way we can structure information in clear tabs. 
With the current implementation there is a lot going on on one screen especially when we want to make things more friendly to use like [this issue](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/118)

We can move the configuration details and deploy status to one tab and leave operations and the schema comparison closer together for example and have space for future additions.

_edit: I've read over the description and refined it to be clearer_

**Comment by tcmorris** [comment]
Agree, tidying up the display of the Deploy dashboard would be welcome. I think this could be done similar to how the Examine dashboard works however, whereby there is a link to view more details and then you can go back to the overview.

**Comment by FransdeJong** [comment]
The examine dashboard lives under settings.
My point is to move deploy to a separate section. This way we can choose who sees the dashboard and who doesn't.
Also it's easier to divide it up in separate screens since you can do whatever you like in a custom section.

I don't really like going to settings, to deploy and then also have to go back and forth between the settings. It feels a bit like a hack. especially since there is already a empty deploy section that is unused.

I'll rename the issue to make this case more clear.

**Comment by ronaldbarendse** [comment]
Thanks for this idea @FransdeJong, and apologies for the long silence on it.

Back in 2022 this wasn't something we could do cleanly. In the AngularJS backoffice, Deploy was a single dashboard under Settings, and giving it a separate section (and separate permissions) would have meant rebuilding most of the UI. Two things have changed since then:
- The CMS backoffice rewrite that started in v14 made sections, sidebar menus, menu items and workspaces plain extension manifests, and section visibility is now driven by the section permissions on the user group.
- In Deploy 18 we moved the Deploy dashboards into separate tree items with their own workspaces (Status, Schema and Configuration). They still live under the Settings section, but each one is a standalone workspace now.

Because of this, moving Deploy into its own section is a small, frontend-only extension.

I've attached a sample that does exactly that: [DeploySection.zip](https://github.com/user-attachments/files/32188970/DeploySection.zip).

To try it:
1. Extract the ZIP into the root of your Umbraco project (it only contains `App_Plugins/DeploySection` with the TypeScript sources, no build output).
2. Run `npm ci` and `npm run build` in `App_Plugins/DeploySection`.
3. Start the site, go to Users > User Groups and add the "Deploy Section" section to the groups that should see it.

What the sample does:
- Registers a `Deploy.Section` section at `/umbraco/section/deploy` that shows Deploy's existing Status, Schema and Configuration menu. The workspaces open inside the new section without any changes to Deploy.
- Requires both the new Deploy section and the Settings section on the user group. Deploy's Management API still authorizes on Settings access, so this keeps the server side working unchanged. A group with only the Deploy section will not see it.
- Hides the Deploy menu from the Settings section. If you'd rather have it in both places, remove the single `exclude` line in `src/entrypoints/entrypoint.ts`.

This is an example to build on rather than a supported part of Deploy: Deploy itself keeps its menu under Settings for now. The `README.md` in the ZIP has the details.

I'm closing this discussion since the original request is now achievable. If the new setup still doesn't cover your specific needs, for example splitting the screens differently or separating permissions further, please open a new discussion so we can look at it against the current backoffice.
